tests/hazmat/backends/test_openssl.py

```diff
@@ -169,6 +169,52 @@ def test_bn_to_int(self):
         bn = backend._int_to_bn(0)
         assert backend._bn_to_int(bn) == 0
 
+    def test_bn_generate_prime(self):
```

Comment:
We don't add tests for our bindings only so you'll probably want to pull these tests into your own codebase.